packages/design-system/src/components/label.tsx

```diff
@@ -0,0 +1,78 @@
+import type { LabelHTMLAttributes, ReactNode, Ref } from "react";
+import { forwardRef } from "react";
+import { styled } from "../stitches.config";
+import { theme } from "../stitches.config";
+
+const StyledLabel = styled("label", {
+  boxSizing: "border-box",
+  flexShrink: 0,
+
+  // shared style
+  py: 2,
```

Comment:
Should use `theme.spacing` unless the comment above explains that this is an exception